### PR TITLE
Add support for extraUrlParams in AT Internet

### PR DESCRIPTION
**Pull request by BenDz**

### CODE REVIEW

**Comment by avimehta** [comment]
The test needs to be updated. rest lgtm.

**Comment by BenDz** [comment]
@avimehta Fixed! Thanks

**Comment by dvoytenko** [comment]
Ping @avimehta

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ export const ANALYTICS_CONFIG = {
       'suffix': '&ref=${documentReferrer}',
       'pageview': '${base}&' +
         'p=${title}&' +
-        's2=${level2}${suffix}',
+        's2=${level2}${extraUrlParams}${suffix}',
```
Comment (avimehta) [inline extensions/amp-analytics/0.1/vendors.js]: just an observation. You are adding extraUrlParams to the pageview request. Would you not want to add it to the beginning of suffix?

**Pull request merged**